shopwareerpconnect/payment_method.py

```diff
@@ -3,7 +3,7 @@
 
 
 class PaymentMethod(models.Model):
-    _inherit = "payment.method"
+    _inherit = "account.payment.mode"

```

Comment:
is the model renamed as such ? Refer here the proof plz

Reply:
It replaces the object "payment.method"
    of the module "sale_payment_method" of OCA/sale-workflow from v8
This is written in "account_payment_mode" module from OCA/bank-account in v9.
And the fields that are defined in "payment.method" in v8 are mostly same in v9 except their model name is replaced with "account.payment.mode"